CHANGELOG

```diff
@@ -1,3 +1,7 @@
+#3.0.20
```

Comment:
to follow semver, it should be a minor bump, ad we added a feature.

3.1.0 would be the correct version